data_structures/src/chain.rs

```diff
@@ -2776,6 +2778,8 @@ pub struct ChainState {
     pub alt_keys: AltKeys,
     /// Current superblock state
     pub superblock_state: SuperBlockState,
+    /// Last received Beacons
+    pub last_received_beacons: Vec<(SocketAddr, Option<LastBeacon>)>,
```

Comment:
This field should be in `ChainManager`, not in `ChainState`, because `ChainState` is stored in the database and we cannot simply add new fields.

Reply:
done